### ISSUE TITLE
Autogenerated index.html not including static/style.css

### ISSUE THREAD

**Issue by LordZane** [koute/cargo-web#99]

**Comment by limira** [comment]
I don't think `cargo-web` should generate more than a simple `index.html` like what it do right now! It requires more work in `cargo-web` but not produce any actual benefit. If you have a `styles.css`, just create your own `index.html` to include your `.css`. You just do this only one time for a project. It is not a very difficult or boring-repetitive task to require a tool.

**Comment by koute** [comment]
@LordZane Could you explain in more detail what exactly would you want here? What purpose such a CSS file would serve? What would be in it?

**Comment by LordZane** [comment]
I meant that when cargo-web generated an index.html for me, I expected it to include the style.css file I had in the static folder, not just copy the style.css over. If you don't want to add support for that, it's fine, but it just deviates from what I expected to occur

**Comment by koute** [comment]
You want it to take the `style.css` and paste it inside the `index.html` inline?

**Comment by LordZane** [comment]
No, just add `<link rel="stylesheet" type="text/css" href="style.css" />` to the html. Im saying that since style.css is automatically copied when doing cargo web deploy, I expected the generated html file to include it

**Comment by untitaker** [comment]
I don't think this is a good idea. In my case I want to run a minifier over my CSS and only link the result of that. It would be very confusing if cargo-web were to auto-include random stylesheets (even though it's unlikely that i'd have the source files in `./static/` then)
